BitwardenShared/UI/Platform/Application/Support/Images.xcassets/Icons/TabBar/vault-idle.imageset/Contents.json

```diff
@@ -0,0 +1,16 @@
+{
+  "images" : [
+    {
+      "filename" : "vault-idle-light.pdf",
+      "idiom" : "universal"
+    }
+  ],
```

Comment:
Yeah good question.  I went with one image approach, rendered as template and then let the color drive the state of the icon.  This code already existed in `UI.swift` so i got it for free.  Does that make sense?  It felt better since I didn't need to add more icons.

https://github.com/bitwarden/ios/blob/f588bd5d24d67f636a89bc2c44983c06f93787e3/BitwardenShared/UI/Platform/Application/Appearance/UI.swift#L92C1-L93C87

Reply:
oh maybe i do have the wrong colors!

Reply:
Ah, ok I didn't catch that. It still might be better to use the same approach for both sets of icons, just to be consistent.

Reply:
yeah i can see the pros to that for sure... i'll make the change so its not confusing moving forward.  It makes sense to remove setting the color of the tab bar in `UI.swift` then, right?

Reply:
Does removing those colors affect the text color? It might still be needed for that if we don't specify it elsewhere.

Reply:
oh yeah duh, we need it for the text!  thanks!